Tone/core/clock/Ticker.ts

```diff
@@ -121,9 +127,9 @@ export class Ticker {
 		return this._updateInterval;
 	}
 	set updateInterval(interval: Seconds) {
-		this._updateInterval = Math.max(interval, 128 / 44100);
+		this._updateInterval = Math.max(interval, this._minimumUpdateInterval);
 		if (this._type === "worker") {
-			this._worker.postMessage(Math.max(interval * 1000, 1));
+			this._worker?.postMessage(this._updateInterval * 1000);
```

Comment:
very minor, but the `!` in line 39 `private _worker!: Worker;` asserts that `_worker` is not null, so probably don't need this `?` null check here. Unless the compiler is complaining, i think this line could just be `this._worker.postMessage(this._updateInterval * 1000);`